tests/utils/utils.go

```diff
@@ -158,7 +186,7 @@ func constructAndSendTransaction(
 	numSigners, err := signedMessage.Signature.NumSigners()
 	Expect(err).Should(BeNil())
 
-	gasLimit, err := teleporter.CalculateReceiveMessageGasLimit(numSigners, teleporterMessage.RequiredGasLimit)
+	gasLimit, err := teleporter.CalculateReceiveMessageGasLimit(numSigners, requiredGasLimit)
```

Comment:
do we plan on leavint the dependency on `awm-relayer` for calculating receiving gas limit?

Reply:
also maybe add the receipt checking in the utils function

Reply:
We can consider moving this to `teleporter` as well.